GSG-K3/urban-calendar
# Update/nodemailer

### Content :octocat: 
- this PR mainly fixed this issue #85 due to non installed nodemailer package.
---
### Testing
Write these following commands down in your terminal :musical_keyboard: 
- pull origin update/nodemailer
- npm run dev
- go to http://localhost:3000/reservation on your browser.
- After you fill all the information and submitting the form 
-  you should receive an email by now
